runtests.py

```diff
@@ -2387,9 +2317,6 @@ def main():
     parser.add_option("--no-refnanny", dest="with_refnanny",
                       action="store_false", default=True,
                       help="do not regression test reference counting")
-    parser.add_option("--no-fork", dest="fork",
-                      action="store_false", default=True,
-                      help="do not fork to run tests")
```

Comment:
Not sure if it's good to remove it entirely. Removing the code really just always enables this option, so scripts that pass it will work fine. We could keep the option, add `deprecated=True`, and add a short note in the `help` string that it's useless now.

Reply:
Done (although `deprecated` isn't in `optparse` I think)